### PR TITLE
WIP: Correct all typos

### PR DESCRIPTION
**Pull request by shmup**
Having fun with this one. I'm using this tool: https://github.com/crate-ci/typos

It's a very manual approach. Not letting the tool correct anything.

Generally it's comments being corrected, in much less cases it's a variable, and in one case so far it's a filename.

### CODE REVIEW

File: libc/nt/master.sh [hunk 1]
```diff
@@ -4828,7 +4828,7 @@ imp	'NtInitializeRegistry'					NtInitializeRegistry					ntdll		380
 imp	'NtInitiatePowerAction'					NtInitiatePowerAction					ntdll		381
 imp	'NtIsProcessInJob'					NtIsProcessInJob					ntdll		382
 imp	'NtIsSystemResumeAutomatic'				NtIsSystemResumeAutomatic				ntdll		383
-imp	'NtIsUILanguageComitted'				NtIsUILanguageComitted					ntdll		384
```
Comment (jart) [inline libc/nt/master.sh:4831]: Please revert changes to this file. This is the Windows API. We can't correct typos here.

File: libc/nt/ntdll/NtIsUILanguageComitted.s [hunk 2]
```diff
@@ -1,2 +1,2 @@
 .include "o/libc/nt/ntdllimport.inc"
-.ntimp	NtIsUILanguageComitted
```
Comment (jart) [inline libc/nt/ntdll/NtIsUILanguageComitted.s:2]: Please revert changes to files in this folder.

File: libc/nt/pedef.internal.h [hunk 3]
```diff
@@ -66,7 +66,7 @@
 #define kNtImageFileExecutableImage      0x0002
 #define kNtImageFileLineNumsStripped     0x0004
 #define kNtImageFileLocalSymsStripped    0x0008
-#define kNtImageFileAggresiveWsTrim      0x0010
```
Comment (jart) [inline libc/nt/pedef.internal.h:69]: Probably revert changes to libc/nt in general

File: third_party/tidy/parser.c [hunk 4]
```diff
@@ -4555,9 +4555,9 @@ Node* TY_(ParsePre)( TidyDocImpl* doc, Node *pre, GetTokenMode ARG_UNUSED(mode)
 
                     /*
                       This is basically what Tidy 04 August 2000 did and far more accurate
-                      with respect to browser behaivour than the code commented out above.
+                      with respect to browser behavior than the code commented out above.
```
Comment (jart) [inline third_party/tidy/parser.c:4558]: Please keep the British english.

File: third_party/zip/zipcloak.c [hunk 5]
```diff
@@ -634,19 +634,19 @@ int main(argc, argv)
...
 
     if (decrypt == 0) {
         if (getp("Verify password: ", verify, IZ_PWLEN+1) == NULL)
-               ziperr(ZE_PARMS,
+               ziperr(ZE_PARAMS,
                       "stderr is not a tty (you may never see this message!)");
 
         if (strcmp(passwd, verify))
-               ziperr(ZE_PARMS, "password verification failed");
+               ziperr(ZE_PARAMS, "password verification failed");
```
Comment (jart) [inline third_party/zip/zipcloak.c:646]: Could you revert the change to `PARMS`? It changes too much to be a typo.

**Review by jart — CHANGES_REQUESTED** [review]

File: third_party/zip/zipcloak.c (hunk 5 above)
Comment (shmup) [reply third_party/zip/zipcloak.c:646]: I appreciate all your input across these files. I'll update everything accordingly

File: third_party/tidy/parser.c (hunk 4 above)
Comment (shmup) [reply third_party/tidy/parser.c:4558]: Noted, I'm going to fix the spelling to the correct British spelling though. Still a typo
